retis/src/process/cli/print.rs

```diff
@@ -26,6 +26,9 @@ pub(crate) struct Print {
     #[arg(long, help = "Format used when printing an event.")]
     #[clap(value_enum, default_value_t=CliDisplayFormat::MultiLine)]
     pub(super) format: CliDisplayFormat,
+    #[arg(long, help = "Time format used when printing an event.")]
+    #[clap(value_enum, default_value_t=CliTimeFormat::MonotonicTimestamp)]
```

Comment:
nit: I don't see users typing such a long cli option "--time-format=monotonic-offset" (plus, the word "monotonic" might not mean a lot to many). From a purely user experience pov, how about a flag `--utc` to achieve the same?

Reply:
The initial reasoning was to allow other variant of the time format (as a side note I would not expect anyone to set `--time-format=monotonic-offset` as it's the default value). But I'm also not sure what would be those other time formats :)

I think using `--utc` is fine and better for UX.

Reply:
Do you also suggest to remove `TimeFormat` in `DisplayFormat` and use a simple `utc: bool`?